api/src/models/young.js

```diff
@@ -648,6 +648,29 @@ const Schema = new mongoose.Schema({
     },
   },
 
+  //Phase 0 classe engagée
+  classeEngageeId: {
```

Comment:
Comme le modèle c'est `classe` et non `classeEngagee`, gardons la cohérence.

```suggestion
  classeId: {
```